roles/calico/defaults/main.yaml

```diff
@@ -4,7 +4,11 @@ etcd_endpoints: "{{ hostvars[groups.oo_first_master.0].openshift.master.etcd_url
 
 cni_conf_dir: "/etc/cni/net.d/"
 cni_bin_dir: "/opt/cni/bin/"
+cni_url: "https://github.com/containernetworking/cni/releases/download/v0.4.0/cni-amd64-v0.4.0.tgz"
 
 calico_etcd_ca_cert_file: "/etc/origin/calico/calico.etcd-ca.crt"
 calico_etcd_cert_file: "/etc/origin/calico/calico.etcd-client.crt"
 calico_etcd_key_file: "/etc/origin/calico/calico.etcd-client.key"
+
```

Comment:
Though I wouldn't ever _recommend_ running mismatched versions of calico-ipam and calico-cni,  the API's are fairly stable and shouldn't complain if you do.